skops/io/_sklearn.py

```diff
@@ -163,5 +167,15 @@ def _DictWithDeprecatedKeys_get_instance(state, src):
     (LossFunction, sgd_loss_get_instance),
     (Tree, Tree_get_instance),
     (Bunch, bunch_get_instance),
-    (_DictWithDeprecatedKeys, _DictWithDeprecatedKeys_get_instance),
 ]
+
+# TODO: remove once support for sklearn<1.2 is dropped.
+# Starting from sklearn 1.2, _DictWithDeprecatedKeys is removed as it's no
+# longer needed for GraphicalLassoCV, see #187.
+if _DictWithDeprecatedKeys is not None:
+    GET_STATE_DISPATCH_FUNCTIONS.append(
+        (_DictWithDeprecatedKeys, _DictWithDeprecatedKeys_get_state)
```

Comment:
should also add the same note to `_DictWithDeprecatedKeys_get_state` and `_DictWithDeprecatedKeys_get_instance`